Documentation/faq.md

```diff
@@ -78,6 +78,10 @@ On the other hand, if the downed member is removed from cluster membership first
 
 etcd sets `strict-reconfig-check` in order to reject reconfiguration requests that would cause quorum loss. Abandoning quorum is really risky (especially when the cluster is already unhealthy). Although it may be tempting to disable quorum checking if there's quorum loss to add a new member, this could lead to full fledged cluster inconsistency. For many applications, this will make the problem even worse ("disk geometry corruption" being a candidate for most terrifying).
 
+### Why does etcd lose its leader from disk latency spikes?
+
```

Comment:
Can we add a sentence "This behavior is an intentional design." or similar? then we can explain the why part.